de.dlr.sc.virsat.apps.test/src/de/dlr/sc/virsat/apps/api/external/ModelAPIAppsTest.java

```diff
@@ -86,6 +101,70 @@ public void testGetConcept() {
 		assertNull("Model API gives null for unknown fqn", modelAPI.getConcept(conceptFqnUnknwon));
 	}
 	
+	@Test
+    public void testAddUserToDiscipline() {
+        // Create a sample Discipline
+        Discipline discipline = RolesFactory.eINSTANCE.createDiscipline();
+        discipline.setName("System");
+
+        //Add the created discipline in the discipline list
+        repository.getRoleManagement().getDisciplines().add(discipline);
+        
+        // Set the user name property
+        System.setProperty("user.name", "SampleUser");
+
+        //Add the user to the user's list
+        discipline.getUsers().add("SampleUser");
+        // Call the method that contains the new line
+        modelAPI.initializeRepositoryResource();
+
+        // Verify that the Discipline is added to Disciplines
+        assertTrue("Discipline should be added to Disciplines", repository.getRoleManagement().getDisciplines().contains(discipline));
+
+        // Verify that the user is added to the Discipline
+        assertTrue("User should not be added to Discipline", discipline.getUsers().contains("SampleUser"));
```

Comment:
I think the "not" can be removed in the string?

Reply:
> I think the "not" can be removed in the string?

Thanks 👍. I removed the "not".